packages/web-client/app/helpers/config.ts

```diff
@@ -0,0 +1,18 @@
+import { getOwner } from '@ember/application';
+import Helper from '@ember/component/helper';
+import { get } from '@ember/object';
+
+export default class extends Helper {
+  config: Config | undefined;
+
+  constructor() {
+    super(...arguments);
+
+    this.config = getOwner(this).resolveRegistration('config:environment');
+  }
+
+  compute([path]: [string]) {
+    // @ts-ignore FIXME, can’t figure it out right now
+    return get(this.config, path);
```

Comment:
I would be fine with `return get(this.config as any, path);`. Not sure if typescript would help us here given it's just a getter to be consumed in templates 🤔